src/scss/_variables.scss

```diff
@@ -882,6 +882,7 @@ $form-feedback-icon-invalid: str-replace(
 
 $form-label-font-size: $h4-font-size !default;
 $form-label-font-weight: var(--#{$prefix}font-weight-medium) !default;
+$form-check-input-indeterminate-bg-color: var(--#{$prefix}primary) !default;
```

Comment:
better way is to add it to `$form-check-input-XXX` section (around lines 852)

Reply:
Placed it in line no. 852 in the 2nd commit